src/mockserver.js

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs';
+
+import { logger } from '@user-office-software/duo-logger';
+import { mockServerClient } from 'mockserver-client';
+
+async function mockserver() {
+    var callback = function (request) {
+        if (request.method === 'POST') {
+            const name = String(request.body.xml);
+            let regexp = '<tns:(.*?)>';
+            const test = name.match(regexp);
+            const method = test[1];
+
+            if (
+                name.includes('getBasicPersonDetailsFromUserNumber') ||
+                name.includes('getSearchableBasicPeopleDetailsFromUserNumbers')
+            ) {
+                let calls = '';
+                name.includes('getSearchableBasicPeopleDetailsFromUserNumbers')
+                    ? (calls = '<UserNumbers>(.*?)<')
+                    : (calls = '<UserNumber>(.*?)<');
+                const match = name.match(calls);
+
+                const file = JSON.parse(
+                    fs.readFileSync(
+                        'src/responses/user/' + method + '/' + match[1] + '.txt',
+                        'utf8'
+                    )
+                );
+                const request1 = file.body.xml;
+
+                return {
+                    body: request1,
+                };
```

Comment:
In all mocked responses, we're going to read a file and return its contents. To avoid duplication, we could:
- Declare `file` before the `if` statements begin
- Assign `file` to the appropriate value in each `if` statement, without returning anything
- After all the `if`s, check if `file` is still epty and assign it to the default value if so
- Finally, return the XML value of `file`, which is now pointing at the right file for the request

This should avoid having to write the same `return` statement in all `if` statements.